TheInternet_Pages/WelcomePage.cs

```diff
@@ -0,0 +1,28 @@
+﻿using Apache.NMS;
+using OpenQA.Selenium;
+
+namespace Welcome
+{
+    public class WelcomePage
+    {
+        private readonly IWebDriver _driver;
+        private readonly By _formAuthLink = By.XPath("//*[@id='content']//*[text()='Form Authentication']");
+        private readonly By _checkboxesLink = By.XPath("//*[@id='content']//*[text()='Checkboxes']");
+        private readonly By _dropdownLink = By.XPath("//*[@id='content']//*[text()='Dropdown']");
+
+        public WelcomePage(IWebDriver driver)
+        {
+            _driver = driver;
+            if (!driver.Title.Equals("The Internet"))
+            {
+                throw new IllegalStateException("This is not The Internet Page," + " current page is: " + driver.Url);
+            }
+        }
+
+        public void DisplayLoginPage() =>_driver.FindElement(_formAuthLink).Click();
```

Comment:
maybe Open...Page()?

Reply:
OK, Open Page instead of Display Page everywhere